### PR TITLE
Added Created HTTP response code for Role and User post methods

### PR DESCRIPTION
**Pull request by b0**
When creating Role or User api will return Location header with uri for new entity and json dto with id for created entity...

### CODE REVIEW

File: src/Skoruba.IdentityServer4.Admin.Api/Controllers/RolesController.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ namespace Skoruba.IdentityServer4.Admin.Api.Controllers
     [Route("api/[controller]")]
     [ApiController]
     [TypeFilter(typeof(ControllerExceptionFilterAttribute))]
-    [Produces("application/json")]
+    [Produces("application/json", "application/problem+json")]
```
Comment (xmichaelx) [inline src/Skoruba.IdentityServer4.Admin.Api/Controllers/RolesController.cs]: This is something new to me, but apparently it's a nice way of hinting user that something is wrong and output will be in JSON. I've read about it here: https://symfonycasts.com/screencast/rest/application-problem

File: src/Skoruba.IdentityServer4.Admin.Api/Controllers/RolesController.cs [hunk 2]
```diff
@@ -85,16 +85,19 @@ public async Task<ActionResult<TRolesDto>> Get(string searchText, int page = 1,
...
+        [ProducesResponseType(201)]
+        [ProducesResponseType(400)]
+        public async Task<ActionResult<TRoleDto>> Post([FromBody]TRoleDto role)
         {
             if (!EqualityComparer<TRoleDtoKey>.Default.Equals(role.Id, default))
             {
                 return BadRequest(_errorResources.CannotSetId());
             }
+ 
+            var (identityResult, roleId) = await _identityService.CreateRoleAsync(role);
```
Comment (xmichaelx) [inline src/Skoruba.IdentityServer4.Admin.Api/Controllers/RolesController.cs]: Nice tuples :)

**Comment by skoruba** [comment]
I will add new project for tests regarding to API #383

**Comment by b0** [comment]
Would you merge this since you are ok with it?

Also, is there a "chat room" with two of you we can discuss further developments and goals? Maybe I can help.

Thank you

**Comment by skoruba** [comment]
I will test it, it would be nice implement same login into all methods for creation of some entity. What do you think?

**Comment by maulik-modi** [comment]
@b0, would it be possible for you to apply same logic into all methods for creation of some entity ?

**Comment by b0** [comment]
@skoruba I have added 201 response with Location header for all controllers post methods where applicable.
Feel free to review.

**Comment by skoruba** [comment]
@b0 - great work! I will test it. 👍🏼😊

**Pull request merged**